src/app/routes/utils/constructPageFetchUrl/index.test.ts

```diff
@@ -96,6 +96,9 @@ describe('constructPageFetchUrl', () => {
     ${LIVE_PAGE}    | ${null}         | ${null}    | ${'local'}  | ${'c0000000000t'}                                     | ${'http://localhost/api/local/ukrainian/live/c0000000000t'}
     ${LIVE_PAGE}    | ${null}         | ${null}    | ${'test'}   | ${'c0000000000t'}                                     | ${'https://mock-bff-path/?id=c0000000000t&service=ukrainian&pageType=live&serviceEnv=test'}
     ${LIVE_PAGE}    | ${null}         | ${null}    | ${'live'}   | ${'c0000000000t'}                                     | ${'https://mock-bff-path/?id=c0000000000t&service=ukrainian&pageType=live&serviceEnv=live'}
+    ${LIVE_PAGE}    | ${'arabic'}     | ${null}    | ${'local'}  | ${'67574192'}                                         | ${'https://mock-bff-path/?id=%2Farabic%2Flive%2F67574192&service=arabic&pageType=live&serviceEnv=local'}
+    ${LIVE_PAGE}    | ${'arabic'}     | ${null}    | ${'test'}   | ${'67574192'}                                         | ${'https://mock-bff-path/?id=%2Farabic%2Flive%2F67574192&service=arabic&pageType=live&serviceEnv=test'}
+    ${LIVE_PAGE}    | ${'arabic'}     | ${null}    | ${'live'}   | ${'67574192'}                                         | ${'https://mock-bff-path/?id=%2Farabic%2Flive%2F67574192&service=arabic&pageType=live&serviceEnv=live'}
```

Comment:
I have changed the pathnames to only include the id which is just the numbers and not the /arabic/live/ substring, because in the real code the pathname is passed into constructPageFetchUrl as just the numbers. We want the rest of the substring added _within_  the constructPageFetchUrl function for the bff data fetch url, but I haven't yet traced down the reason for the numbers part of the id being passed into getPageData alone. Perhaps the numbers part of the id is used alone elsewhere (e.g routing).

Reply:
Could we add a couple of additional scenarios for variants? Ideally for both types of Live Page (Tipo & CPS). We might need to check with the Content Team about what the expected URL of a Tipo Live page would be (i.e. is the variant at the end or in the middle of the URL?).